### ISSUE TITLE
Running the editor 

### ISSUE THREAD

**Issue by razvanab** [GaijinEntertainment/DagorEngine#99]
Is there a way to run the editor with the " east_district " sample without compiling the engine?

**Comment by NicSavichev** [comment]
No, **east_district** is not available as source data at the moment, only as prebuilt resources/location.

**Comment by VeresBalazsLaszlo** [comment]
> Is there a way to run the editor with the " east_district " sample without compiling the engine?

I don't know what's your goal but: https://forum.enlisted.net/en/t/custom-level-bins-for-enlisted/154880

You can fly around the scene in the editor.

**Comment by NicSavichev** [comment]
> Is there a way to run the editor with the " east_district " sample without compiling the engine?

Don't know what editor do you need but you can switch to embedded (daEditorE) with F12 key. Here you can inspect and move ECS entities of scene.

**Comment by razvanab** [comment]
I see. Thank you.
